.github/workflows/ci-build-check.yaml

```diff
@@ -13,4 +13,3 @@ jobs:
         run: |
           chmod +x gradlew
           ./gradlew build
-          
```

Comment:
I can revert this if its a problem, I was playing around with ci but I didn't revert it properly